<!-- -->
<launch>

  <node pkg="geometric_terrain_classifier" type="geometric_classifier" name="geometric_classifier" output="screen">

    <remap from="/points_raw" to="/surfel_map/pointcloud" />

  </node>
  
</launch>
